msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -1,520 +1,35 @@
-    ```
-
-  * **Target Schema**: `Container`
-
-    ```json
-        {
-                  "type": "Container",
-                  "$data": "${$root}",
-                  "items": [
-                    {
-                      "type": "TextBlock",
-                      "text": "id: ${if(id, id, 'N/A')}",
-                      "wrap": true
-                    },
-                    {
-                      "type": "TextBlock",
-                      "text": "category.name: ${if(category.name, category.name, 'N/A')}",
-                      "wrap": true
-                    }
-                  ]
-                }
-                
-    ```
-
-  </details>
-
-* `object`: An object is converted to a nested property in Adaptive Card.
-
-  <details><summary>Example</summary>
-
-  * **Source Schema**: `object`
-
-    ```yml
-    components:
-      schemas:
-        Pet:
-            category:
-              type: object
-            properties:
-              id:
-                type: integer
-              name:
-                type: string
-
-    ```
-
-  * **Target Schema**: Nested property in an Adaptive Card
-
-    ```json
-    {
-      "type": "TextBlock",
-      "text": "category.id: ${if(category.id, category.id, 'N/A')}",
-      "wrap": true
-    },
-    {
-      "type": "TextBlock",
-      "text": "category.name: ${if(category.name, category.name, 'N/A')}",
-      "wrap": true
-    }
-
-    ```
-
-  </details>
-
-* `image`: If a property is an image URL, then it converts to an Image element in the Adaptive Card.
-
-  <details><summary>Example</summary>
-
-  * **Source schema**: `image`
-
-    ```yml
-        image:
-          type: string
-          format: uri
-          description: The URL of the image of the item to be repaired
-
-    ```
-
-  * **Target Schema**: `"Image"`
-
-    ```json
-    {
-          "type": "Image",
-          "url": "${image}",
-          "$when": "${image != null}"
-        }
-
-    ```
-
-  </details>
-
-</details>
-
-## Authentication
-
-You can implement authentication in API-based message extensions to provide secure and seamless access to applications. If your message extension requires authentication, add the `authorization` property under `composeExtensions` in app manifest and define the type of authentication for your application by setting the `authType` property under `authorization`. To enable authentication for your message extension, update your app manifest with any of the following authentication methods:
-
-<details><summary id="none">none</summary>
-<br>
-
-You can update `none` as a value for `authorization` in an API-based message extension when the message extension doesn't require any authentication for the user to access the API.
+Update `none` as a value for `authorization` in an API-based message extension when the API doesn't require any authentication for the user. When Teams service sends a request to the API, it doesn't supply any authentication information.
```

Comment:
We have other sections in respective files [Secret service auth](https://review.learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/api-based-secret-service-auth?branch=pr-en-us-10743) and [Microsoft Entra](https://review.learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/api-based-microsoft-entra?branch=pr-en-us-10743&tabs=token-v2).

Since this is only a small section , I've included this as part of the overview.